src/config/ConfigService.js

```diff
@@ -40,18 +40,13 @@ export default class ConfigService extends Service {
 
 
 	initialize() {
-		let config_url = this.App.Config.get('CONFIG_PATH');
-
-		// Provide backward compatibility with CONFIG_URL
-		if (this.App.Config.get('CONFIG_URL')) {
-			config_url = this.App.Config.get('CONFIG_URL');
-		}
+		const dynamic_config_url = document.getElementsByName('x-config')[0]?.content;
```

Comment:
Can you add as a comment the example of the tag used for specification of the documentation? @titkov49 

Also this needs to be documented somewhere ...

Reply:
@ateska 
Done: https://github.com/TeskaLabs/asab-webui/pull/149